airmozilla/manage/archiver.py

```diff
@@ -18,7 +18,7 @@
 def auto_archive(verbose=False):
     events = (
         Event.objects
-        .filter(status=Event.STATUS_PENDING,
+        .filter(status=Event.STATUS_PROCESSING,
```

Comment:
No, this needs to be:
```python
.filter(Q(status=Event.STATUS_PENDING) | Q(status=Event.STATUS_PROCESSING))
```
You can find examples of using `Q` in other places of the code. 
Basically, what this would do is make it an OR statement. 

Some events (e.g. uploaded pre-recorded files) will go straight into PENDING and those that *were* live will go into PROCESSING status.